frontend/packages/ceph-storage-plugin/src/components/create-backingstore-page/create-bs-page.tsx

```diff
@@ -23,8 +28,12 @@ const CreateBackingStoreFormPage: React.FC<CreateBackingStoreFormPageProps> = ({
         <BreadCrumbs
           breadcrumbs={[
             {
-              name: 'Openshift Container Storage',
-              path: resourcePathFromModel(ClusterServiceVersionModel, appName, ns),
+              name: isODF ? 'OpenShift Data Foundation' : 'OpenShift Container Storage',
+              path: isODF ? '/odf' : resourcePathFromModel(ClusterServiceVersionModel, appName, ns),
+            },
+            {
+              name: t('ceph-storage-plugin~Backing Store'),
+              path: isODF ? `/odf/resource/${referenceForModel(NooBaaBackingStoreModel)}` : null,
             },
```

Comment:
Sorry for the trouble after talking to QE they don't think they want this middle step. So please remove them. Thanks.